Gemfile.lock

```diff
@@ -347,33 +342,33 @@ GEM
       netrc (~> 0.8)
     rspec-core (3.8.2)
       rspec-support (~> 3.8.0)
-    rspec-expectations (3.8.4)
+    rspec-expectations (3.8.6)
       diff-lcs (>= 1.2.0, < 2.0)
       rspec-support (~> 3.8.0)
-    rspec-mocks (3.8.1)
+    rspec-mocks (3.8.2)
       diff-lcs (>= 1.2.0, < 2.0)
       rspec-support (~> 3.8.0)
-    rspec-rails (3.8.2)
+    rspec-rails (3.8.3)
```

Comment:
### [rspec-rails](https://github.com/rspec/rspec-rails)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 3.8.2 → 3.8.3

[_[change-log](https://github.com/rspec/rspec-rails/blob/v3.8.3/Changelog.md), [source-code](https://github.com/rspec/rspec-rails)_]





<details>
<summary>Commits</summary>

A change of **30** commits. See the full changes on [the compare page](https://github.com/rspec/rspec-rails/compare/v3.8.2...v3.8.3).

These are the first 10 commits:
- (a2e9de6) [switch git from git to https (#1697)](https://github.com/rspec/rspec-rails/commit/a2e9de6289a58ebedb154fd527c168f819754c5e)
- (a4fad1a) [Prevent Github complaining about FFI / Rubyzip (#2040)](https://github.com/rspec/rspec-rails/commit/a4fad1a93266b7cbd3a146b36e978b2bc80987eb)
- (a9094cb) [Pin sqlite to avoid build issue (#2078)](https://github.com/rspec/rspec-rails/commit/a9094cb20687889bf57c6dbaa624a44fe46446d8)
- (d18a7f2) [Fix fixtures path for model generator when using namespaced …](https://github.com/rspec/rspec-rails/commit/d18a7f2bc27531bfbb97313da02361e1c2a6badd)
- (2adf996) [Changelog for #2077](https://github.com/rspec/rspec-rails/commit/2adf9964cb34b60e6818fedf02549260227b7924)
- (26611ad) [Improve `errors` method check in be_valid matcher (#2096)](https://github.com/rspec/rspec-rails/commit/26611add1d8d3164190031efe9ea0eeba2753c10)
- (5bd697c) [Changelog for #2096](https://github.com/rspec/rspec-rails/commit/5bd697c0e2dcd722586afc9baceb034a0c872007)
- (e4bf5b4) [Update ISSUE_TEMPLATE.md](https://github.com/rspec/rspec-rails/commit/e4bf5b4001e7c54befb2bc6fd3fea37be5d29db1)
- (febef9d) [Updated travis build scripts (from rspec-dev) (#2105)](https://github.com/rspec/rspec-rails/commit/febef9dcf53affc6c695de8c10ddb40e2ea505b9)
- (85fa2ef) [Turn value into a String before regex match (#2106)](https://github.com/rspec/rspec-rails/commit/85fa2ef24fbefe868606b0abeff80099e3e3329e)


</details>